components/tikv_util/src/lru.rs

```diff
@@ -314,8 +314,8 @@ where
         if new_cap == 0 {
             new_cap = 1;
         }
-        if new_cap < self.capacity && self.map.len() > new_cap {
-            for _ in new_cap..self.map.len() {
+        if new_cap < self.capacity && self.size() > new_cap {
```

Comment:
I think units of `self.size()` (bytes) and `new_cap` (map entry count) are different.

Reply:
But we use the total sizie instead of the entry count in the old value cache?

```
+------+-----------------+----------------------------------+----------+
| Type | Instance        | Name                             | Value    |
+------+-----------------+----------------------------------+----------+
| tikv | 127.0.0.1:20160 | cdc.old-value-cache-memory-quota | 10244444 |
+------+-----------------+----------------------------------+----------+
```

Reply:
The capacity of a cache should generally refer to the size(bytes), not the number of entries in the cache.

Reply:
Got it!